From: Chocobo1 <Chocobo1@users.noreply.github.com>
diff --git a/psymodel.c b/psymodel.c

--- a/psymodel.c
+++ b/psymodel.c
@@ -273,7 +273,7 @@ static const int tab_mask_add_delta[] = { 2, 2, 2, 1, 1, 1, 0, 0, -1 };
 inline static int
 mask_add_delta(int i)
 {
-    STATIC_ASSERT_EQUAL_DIMENSION(tab_mask_add_delta,tab);
+    //STATIC_ASSERT_EQUAL_DIMENSION(tab_mask_add_delta,tab);
     assert(i < (int)dimension_of(tab));
     return tab_mask_add_delta[i];
 }
